e2e/testcases/birth/1-birth-event-declaration.spec.ts

```diff
@@ -1,54 +1,541 @@
-import { expect, test } from '@playwright/test'
-import { createPIN, login } from '../../helpers'
+import { expect, test, type Page } from '@playwright/test'
+import { createPIN, getToken, login } from '../../helpers'
+import { createDeclaration } from './helpers'
 
-test.describe('1. Birth event declaration', () => {
-  test.beforeEach(async ({ page }) => {
-    await login(page, 'k.mweene', 'test')
-    await createPIN(page)
+import TEST_DATA_1 from './data/1-both-mother-and-father.json'
+import faker from '@faker-js/faker'
+
+test.describe.serial('1. Birth event declaration', () => {
+  let page: Page
```

Comment:
@jamil314 Can we do it so that steps 1.1 - 1.9 are inside of one `test.describe` block, that is configured to be serial

```ts
test.describe.serial('group', () => {
  test('1.1', async ({ page }) => {});
  test('1.2', async ({ page }) => {});
});
```

All other tests could be then done so that it logs in, opens the form, and then for instance tests the exit button or the delete declaration button.